dotcom-rendering/scripts/webpack/server.js

```diff
@@ -55,6 +53,7 @@ module.exports = () => ({
 									},
 								],
 							],
+							compact: true,
```

Comment:
See: https://stackoverflow.com/questions/29576341/what-does-the-code-generator-has-deoptimised-the-styling-of-some-file-as-it-e